src/index.html

```diff
@@ -6,9 +6,94 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+    rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <a
+          href="#home"
+          class="logo"
+        >
+          <img
+          src="./images/logo.png"
+          alt="moyo"
+        >
+        </a>
+        <nav class="nav">
+          <ul class="nav_list">
+            <li class="item">
+              <a
+                class="is-active"
+                href="#apple"
+              >
+                apple
+              </a>
+            </li>
+
+            <li class="item">
+              <a
+                href="#samsung"
+              >
```

Comment:
not critical, but when you have only one short attribute it can be placed on one line with tag
but yes if you have 2-3 or more it will be good as you did
```suggestion
              <a href="#samsung">
```